core/src/main/resources/com/linecorp/armeria/server/docs/assets/js/armeria.js

```diff
@@ -245,7 +251,18 @@ $(function () {
         type: 'POST',
         url: serviceInfo.debugPath,
         data: request,
-        headers: httpHeaders,
+        beforeSend: function (xhr) {
+          for (var name in httpHeaders) {
+            if (httpHeaders.hasOwnProperty(name)) {
+              var values = httpHeaders[name];
+              if (Array.isArray(values)) {
+                xhr.setRequestHeader(name, values.join());
```

Comment:
Just tried by myself and it seems like the previous header value is just replaced, at least in Chrome.